### PR TITLE
docs: fix invalid links

### PR DESCRIPTION
**Pull request by bmihelac**
fixes links on http://docs.hood.ie/en/latest/api/client/hoodie.html page

### CODE REVIEW

File: docs/api/client/hoodie.rst [hunk 1]
```diff
@@ -28,10 +28,10 @@ id, session or the connection status to the backend.
 
 Hoodie integrates Hoodie’s client core modules:
 
--  `The account API <client/hoodie.account.html>`__
--  `The store API <client/hoodie.store.html>`__
--  `The connectionStatus API <client/hoodie.connection-status>`__
--  `The log API <client/hoodie.log.html>`__
```
Comment (gr2m) [inline docs/api/client/hoodie.rst:34]: these links currently work for me on http://docs.hood.ie/en/latest/api/index.html – are you sure we can change them?

Comment (bmihelac) [reply docs/api/client/hoodie.rst:34]: > these links currently work for me on http://docs.hood.ie/en/latest/api/index.html – are you sure we can change them?

This page have proper link, but this one does not (it is one level deeper):
http://docs.hood.ie/en/latest/api/client/hoodie.html

Comment (gr2m) [reply docs/api/client/hoodie.rst:34]: Thanks for the explanation, sorry that I missed it. And thanks for your help 💐

**Review by gr2m — APPROVED** [review]

**Comment by bmihelac** [comment]
no problem, thanks for merging

**Pull request merged**